circleci/circleci.go

```diff
@@ -0,0 +1,34 @@
+package circleci
+
+import (
+	"github.com/airware/vili/log"
+	"github.com/jszwedko/go-circleci"
+)
+
+var config *Config
+var client *circleci.Client
+
+// Config is the circle configuration
+type Config struct {
+	Token   string
+	BaseURL string
+}
+
+// Init Initializes the circle ci client
+func Init(c *Config) error {
+	config = c
+	if c.Token == "" || c.BaseURL == "" {
+		log.Warn("Missing circle ci Token")
+	}
+	client = &circleci.Client{Token: c.Token}
```

Comment:
It could be helpful to add a [Logger](https://godoc.org/github.com/jszwedko/go-circleci#Logger) implementation here which writes messages to our `log` library on calls to `Printf`, perhaps at info level.

Reply:
Added.